src/tests/unit/tests/debug-tools/components/debug-tools-view.test.tsx

```diff
@@ -33,9 +45,17 @@ describe('DebugToolsView', () => {
                 } as UserConfigurationStoreData,
             } as DebugToolsViewState;
 
-            const wrapped = shallow(<DebugTools deps={deps} storeState={storeState} />);
+            const renderResult = render(<DebugTools deps={deps} storeState={storeState} />);
 
-            expect(wrapped.getElement()).toMatchSnapshot();
+            expect(renderResult.asFragment()).toMatchSnapshot();
+            //In snapshot getting undefined, to avoid the undefined value added below if condition.
```

Comment:
It'd be good to add an `else` clause where you test that each component didn't get called to cover all the cases.

Reply:
Added else part.